css/styles.css

```diff
@@ -1,6 +1,6 @@
 .notification {
 	display: block;
-	padding: 7px 15px 50px 15px;
+	padding: 15px 15px 50px 15px;
```

Comment:
Can you change these 15 to 12px? That Is the usual padding we use.

And the bottom-padding should also be 12px, no? @nickvergessen we don’t have a min-height, we have a max-height, and this 50px of padding causes the strange whitespace below a single notification.